doc/modules/ratelimit.md

```diff
@@ -37,22 +43,22 @@ Where `type` is one of:
 Both these attributes are floating point values.
 
 From version `1.5` it is also possible to define limits in a simplified form:
-  bounce_to = "2 / 5m";
-
-Where the first number is number of messages and the second is timeframe. For example, the line above defines a limit for bounce messages per recipient at level 2 message in 5 minutes.
+`bounce_to = [100, "2 / 5m"];`
```

Comment:
But why do we need size in this case? It is illogical, since we already have the size: namely, 2 in this case.